src/dev-app/ff-perf-test/BUILD.bazel

```diff
@@ -0,0 +1,21 @@
+load("//tools:defaults.bzl", "ng_module", "sass_binary")
+
+package(default_visibility = ["//visibility:public"])
+
+ng_module(
+    name = "ff-perf-test",
```

Comment:
Should we change this to `form-field-performance-test`? We've been avoiding abbreviations in names until now.

Reply:
I added this so we can judge whether we're happy with the performance of this solution. Once we're satisfied I'll just remove this commit

Reply:
Re-confirmed the perf results after switching to observables and it still looks good, so dropping this commit now